src/sage/rings/abc.pyx

```diff
@@ -14,13 +14,13 @@ class NumberField_quadratic(Field):
     EXAMPLES::
 
         sage: import sage.rings.abc
-        sage: K.<sqrt2> = QuadraticField(2)                                     # optional - sage.rings.number_field
-        sage: isinstance(K, sage.rings.abc.NumberField_quadratic)               # optional - sage.rings.number_field
+        sage: K.<sqrt2> = QuadraticField(2)                                             # optional - sage.rings.number_field
+        sage: isinstance(K, sage.rings.abc.NumberField_quadratic)                       # optional - sage.rings.number_field
         True
```

Comment:
Why this column change?

Reply:
Just standardizing on column 88, which works well in the furo style

Reply:
I once thought the `# optional` tags could better be placed farther to the right, out of view. Does column 88 work for that?

Reply:
`src/sage/rings/abc.pyx` seems not in the documentation...

Reply:
The placement at column 88 makes "# option" appear, and to see the details, one needs to scroll horizontally. I prefer this over moving them completely out of view.

Reply:
> `src/sage/rings/abc.pyx` seems not in the documentation...

Indeed; I'll fix this here.

Reply:
Done